readthedocs/builds/models.py

```diff
@@ -734,6 +734,14 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('builds_detail', args=[self.project.slug, self.pk])
 
+    def get_full_url(self):
+        """Get full url including domain"""
+        full_url = 'https://{domain}{absolute_url}'.format(
```

Comment:
We could check for DEBUG here and set it to `http` if we wanted something that works in dev, but I'd prefer a better solution. We already do this somewhere else: https://github.com/readthedocs/readthedocs.org/blob/master/readthedocs/api/v3/serializers.py#L31